ideas/313-core-networking.md

```diff
@@ -0,0 +1,70 @@
+---
+id: #313-core-network-incentives
+title: Core Network Incentives Swarm Proposal
+status: research
+contributors:
+    - igor
+    - andrea mp
+budget:
+- actual: xxx
+- estimate: yyy
+- currency: ETH/USD/SNT
+---
+
+# Core Networking Swarm proposal
+
+## Summary and Goal(s)
+
+Research and develop framework and incentivization structure for people
+deploying and using Status nodes according to [the whitepaper](https://status.im/whitepaper.pdf).
+
+### Additional contributors
+- oskart
+- dmitrys
+
+## Communication
+(required)
+`status channel (same as swarm id)`: [#313-core-network-incentives](https://get.status.im/chat/public/313-core-networking)
+`sync frequency`: Weekly Sync Mondays, 11:00 CET
+
+## Research
+(required) 
+`timebox research (approx)` 
+Research deadline: January 31th
+
+(optional)
+`eips`, `competitors`, `existing research`, `wireframes`, `spike (PoC)`
```

Comment:
- https://1337alliance.com/
- 168-paid-masternodes docs (incl micropayments etc)
- whitepaper
- protocol work and leveraging data sync data, eg

Reply:
done